mimic/rest/cinder_api.py

```diff
@@ -31,7 +31,14 @@ def catalog_entries(self, tenant_id):
         """
         return [
             Entry(
-                tenant_id, "volume", "cloudBlockStorage",
```

Comment:
@wallrj @BenjamenMeyer - We should definitely register it as both.

However, I think @wallrj points out an important problem here: the endpoint type is "volume" but the presented API is cinder v2; doesn't that mean it should be "volumev2"?  Honestly I'm not sure what the _right_ way is for OpenStack to tell the user what API to expect; I thought it was "type" but the documentation does not appear to be explicit and the examples seem a little inconsistent on that point.

Regarding API completeness - I think calling it "volunteer based" might be slightly misleading, most Rackspace folks who work on it do so as part of their jobs - but it's not a supported product, it's just something we work on as time allows, in support of other projects :-).  Which is exactly why it's so important that we get outside help to make it more complete - so thanks for that!